bin/_mocha

```diff
@@ -428,7 +428,10 @@ function exit(code) {
   done();
 }
 
-process.on('SIGINT', function() { runner.abort(); })
+process.on('SIGINT', function() {
+  runner.abort();
+  process.exit(130);
```

Comment:
This is going to cause problems, because `runner.abort()` is attempting to exit cleanly.

Instead, you should be able to override `runner.failures`; [see here](https://github.com/boneskull/mocha/blob/5e1cd44c1f4e7301b6368209abf986bd6becca92/lib/runner.js#L817-L817).